src/gateway/shard.rs

```diff
@@ -2,10 +2,11 @@ use crate::constants::{self, close_codes};
 use crate::internal::prelude::*;
 use crate::model::{
     event::{Event, GatewayEvent},
-    gateway::Activity,
+    gateway::{Activity},
```

Comment:
I'm assuming the addition of these braces was an accident?
```suggestion
    gateway::Activity,
```

Reply:
Yes, that was an accident.